app/views/issues/edit.html.erb

```diff
@@ -10,12 +10,11 @@
   </nav>
 <% end %>
 
-<div class="content-container">
-  <div id="issues_editor">
-    <div class="note-text-inner">
-      <h4 class="header-underline">Edit issue</h4>
-      <div class="autofocus" autofocus="true" tabindex="1"></div> <!-- This div is needed to allow correct tabbing order in Chrome with page is loaded with turbolinks -->
-      <%= render 'form' %>
-    </div>
+<div class="content-container" id="issues_editor">
```

Comment:
@MattBudz make sure this change doesn't have any JS implications before merging, I distrust the existence of `id="issues_editor"` and would like to make sure we understand why is it there, and whether it's safe to move it.